CLIENT-SPECIFICATION.md

```diff
@@ -205,20 +202,21 @@ The [`LC_MESSAGES` environment variable](https://www.gnu.org/software/gettext/ma
 
 Here's an example of how the lookup should be done on `linux` having set `LANG=it` and `LANGUAGE="it:fr:en"`:
 
-    1. pages.it/linux/some-page.md  -> does not exist
-    2. pages.fr/linux/some-page.md  -> does not exist
-    3. pages/linux/some-page.md     -> does not exist
-    4. pages.it/common/some-page.md -> does not exist
-    5. pages.fr/common/some-page.md -> does not exist
-    6. pages/common/some-page.md    -> FOUND!
+```md
+  1. pages.it/linux/some-page.md  -> does not exist
+  2. pages.fr/linux/some-page.md  -> does not exist
+  3. pages/linux/some-page.md     -> does not exist
+  4. pages.it/common/some-page.md -> does not exist
+  5. pages.fr/common/some-page.md -> does not exist
+  6. pages/common/some-page.md    -> FOUND!
+```
```

Comment:
See above comment, but I'm still unclear on why a code fence is needed here. If it's to align the `->`, then perhaps we could consider reformatting this as a table?
```suggestion
Step	| Path checked					| Outcome
--------|-------------------------------|-----------------------
1		| pages.it/linux/some-page.md	| does not exist
2		| pages.fr/linux/some-page.md	| does not exist
3		| pages/linux/some-page.md		| does not exist
4		| pages.it/common/some-page.md	| does not exist
5		| pages.fr/common/some-page.md	| does not exist
6		| pages/common/some-page.md		| FOUND!
```

Reply:
Yeah, a table would be better. Will apply your suggestion.